src/NServiceBus.Core.Tests.x86/When_running_under_a_32Bit_process.cs

```diff
@@ -1,22 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.x32
-{
-    using System.Linq;
-    using Hosting.Helpers;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_running_under_a_32Bit_process
-    {
-        [Test]
-        public void Should_skip_x64_assemblies()
```

Comment:
@Particular/nservicebus-maintainers or is this test enough of a reason to keep them?